.github/workflows/coverage.yml

```diff
@@ -18,12 +18,14 @@ jobs:
     if: "!contains(github.event.head_commit.message, '[skip ci]')"
 
     steps:
-    - name: Build Gwion
-      uses: fennecdjay/gwion-action@v1
-      with:
-        dir: .
-        ref: ${{ github.sha }}
-        run: 'true'
+    - name: Checkout
```

Comment:
That's the thing asked for in the issue, you uncovered so many things beynd that!
Thank you.